### PR TITLE
RELEASE-2018-2-20: updated changlog and package-lock file

### PR DESCRIPTION
**Pull request by aaron-martin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,7 +24,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 - functionality to add simple products to the cart
 - functionality to return BigCommerce checkout url
 
-[Unreleased]: https://github.com/shopgate/cloud-ext-bigcommerce-cart/compare/v0.1.2...HEAD
+[Unreleased]: https://github.com/shopgate/cloud-ext-bigcommerce-cart/compare/v0.1.3...HEAD
+[0.1.3]: https://github.com/shopgate/cloud-ext-bigcommerce-cart/compare/v0.1.3...v0.1.3
```
Comment (sofib) [inline CHANGELOG.md]: Compare v0.1.2...v0.1.3

File: extension/package-lock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shopgate/bigcommerce-cart",
-  "version": "0.1.2",
+  "version": "0.1.3",
```
Comment (sofib) [inline extension/package-lock.json]: package.json should be changed as well

**Comment by sofib** [comment]
extension-config.json needs to be updates as well.

File: CHANGELOG.md (hunk 1 above)
Comment (aaron-martin) [reply CHANGELOG.md]: fixed

File: extension/package-lock.json (hunk 2 above)
Comment (aaron-martin) [reply extension/package-lock.json]: updated

**Comment by aaron-martin** [comment]
corrected etension-config.json file.

**Pull request merged**